packages/core/src/services/commits/find.ts

```diff
@@ -0,0 +1,25 @@
+import database from '$core/client'
+import { commits } from '$core/schema'
+import { desc, eq, isNull } from 'drizzle-orm'
+
+export default async function findCommit(uuid?: string, tx = database) {
+  if (uuid === 'HEAD') {
+    return (
+      await tx.select().from(commits).orderBy(desc(commits.id)).limit(1)
+    )[0]
+  }
+
+  return (
+    await tx
+      .select({ id: commits.id })
+      .from(commits)
+      .where(selectCondition(uuid))
+      .limit(1)
+  )[0]
```

Comment:
`tx.query.commits.findFirst({ where: selectCondition(uuid) })`

Reply:
hmmm does not seem to implement orderby